src/test/mir-opt/array_index_is_temporary.main.SimplifyCfg-elaborate-drops.after.32bit.mir

```diff
@@ -36,10 +36,10 @@ fn main() -> () {
         StorageLive(_5);                 // scope 3 at $DIR/array-index-is-temporary.rs:16:12: 16:29
         StorageLive(_6);                 // scope 4 at $DIR/array-index-is-temporary.rs:16:25: 16:26
         _6 = _3;                         // scope 4 at $DIR/array-index-is-temporary.rs:16:25: 16:26
-        _5 = foo(move _6) -> bb1;        // scope 4 at $DIR/array-index-is-temporary.rs:16:21: 16:27
+        _5 = Zst: unsafe fn(*mut usize) -> u32 {foo}(move _6) -> bb1; // scope 4 at $DIR/array-index-is-temporary.rs:16:21: 16:27
```

Comment:
Is it possible to avoid all these changes? They make it harder to read IMO.

Reply:
I am not sure how to do that. The printing code seems to be spread across multiple files. I think I need a new match arm [here](https://github.com/rust-lang/rust/blob/bc5a85750626f01c5bfc43ab9ed62025d6d0d603/compiler/rustc_middle/src/mir/mod.rs#L2751) but I cannot call `print_value_path` there.

Amusingly that function already has a comment "Move that into `mir/pretty.rs`", so I am not the first to run into this problem.^^